appeals/web/src/server/appeals/appeal-details/lpa-questionaire/lpa-questionnaire.mapper.js

```diff
@@ -1,335 +1,190 @@
-		title: 'Listed building',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.isListedBuilding),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-	if (lpaQuestionnaireData.isListedBuilding) {
-		mappedData.listedBuildingDetails = {
-			title: 'Listed building details',
-			value: lpaQuestionnaireData.listedBuildingDetails?.map(
-				(/** @type {{ grade: string; description: string; }} */ item) =>
-					`${item.grade}: ${item.description}`
-			),
-			valueType: 'link',
-			actionText: 'Change',
-			actionLink: '#'
-		};
-	}
-
-	mappedData.doesAffectAListedBuilding = {
-		title: 'Affects a listed building',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.doesAffectAListedBuilding),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.affectsListedBuildingDetails = {
-		title: 'Affects a listed building details',
-		value: lpaQuestionnaireData.affectsListedBuildingDetails?.map(
-			(/** @type {{ grade: string; description: string; }} */ item) =>
-				`${item.grade}: ${item.description}`
-		),
-		valueType: 'link',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.doesAffectAScheduledMonument = {
-		title: 'Affects a scheduled monument',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.doesAffectAScheduledMonument),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.inCAOrrelatesToCA = {
-		title: 'Conservation area',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.inCAOrrelatesToCA),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.conservationAreaMapAndGuidance = {
-		title: 'Conservation area map and guidance',
-		value: lpaQuestionnaireData.documents?.conservationAreaMapAndGuidance,
-		valueType: 'link',
-		actionText: 'Details',
-		actionLink: '#'
-	};
-
-	mappedData.hasProtectedSpecies = {
-		title: 'Protected species',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.hasProtectedSpecies),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.isTheSiteWithinAnAONB = {
-		title: 'Area of outstanding natural beauty',
-		value: convertFromBooleanToYesNo(lpaQuestionnaireData.isTheSiteWithinAnAONB),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
-	};
-
-	mappedData.designatedSites = {
-		title: 'Designated sites',
-		value: lpaQuestionnaireData.designatedSites?.map(
-			(item) => `${item.description} (${item.name})`
-		),
-		valueType: 'text',
-		actionText: 'Change',
-		actionLink: '#'
+export function lpaQuestionnairePage(lpaData, appealData, currentRoute, session) {
+	const mappedLPAQData = initialiseAndMapLPAQData(lpaData, currentRoute);
+	const mappedAppealData = initialiseAndMapAppealData(appealData, currentRoute);
+	const appealType = appealData.appeal.appealType;
+
+	let caseSummary = {
+		type: 'summary-list',
+		noActions: !session.account.idTokenClaims.groups.includes('appeals_case_officer'),
```

Comment:
I think auth checks are done on group id, and should come from config not be hardcoded (this string is what we use for local dev sometimes). This should be `config.referenceData.appeals.caseOfficerGroupId`.

Reply:
We could use the more fine grained permission set in auth.pipes...